webdriver/conftest.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+import pytest
+import webdriver
+
+import util.cleanup as cleanup
```

Comment:
Do you mean the general pattern (e.g. importing the module and defining the fixture here) or the specific aliasing (e.g. `as cleanup`)?